### PR TITLE
:bug: fix incorrect PR title handling

### PR DESCRIPTION
**Pull request by Rozzii**
This commit:
  - Changes the PR title in the PR verifier script to be a constant string in the BASH context.

### CODE REVIEW

**Comment by Rozzii** [comment]
code injection fixed according to test https://github.com/metal3-io/project-infra/actions/runs/12546662618/job/34982853335?pr=941

Issue with the double quotes has been fixed according to https://github.com/metal3-io/project-infra/actions/runs/12546678748/job/34982894750?pr=941

**Comment by Rozzii** [comment]
Not relevant:
/override metal3-ubuntu-e2e-integration-test-main

**Comment by Rozzii** [comment]
this looks even better thanks @kashifest  https://github.com/metal3-io/project-infra/actions/runs/12547150833/job/34984126406?pr=941#step:3:39

File: .github/workflows/pr-verifier.yaml [hunk 1]
```diff
@@ -13,12 +13,12 @@ jobs:
...
-          WIP_REGEX="^\W?WIP\W"
-          TAG_REGEX="^\[[[:alnum:]\._-]*\]"
-          PR_TITLE="${{ github.event.pull_request.title }}"
+          WIP_REGEX='^\W?WIP\W'
+          TAG_REGEX='^\[[[:alnum:]\._-]*\]'
+          PR_TITLE='${{ github.event.pull_request.title }}'
 
           # Trim WIP and tags from title
-          trimmed_title=$(echo "$PR_TITLE" | sed -E "s/$WIP_REGEX//" | sed -E "s/$TAG_REGEX//" | xargs)
+          trimmed_title=$(echo "${PR_TITLE}" | sed -E "s/$WIP_REGEX//" | sed -E "s/$TAG_REGEX//" | xargs)
```
[the remark this answers is not in the document]
Comment (Rozzii) [reply .github/workflows/pr-verifier.yaml]: done

**Comment by Rozzii** [comment]
Not relevant:
/override metal3-ubuntu-e2e-integration-test-main

**Review by dtantsur — APPROVED** [review]

**Pull request merged**